sdk/python/pulumi_azure/ad/get_application.py

```diff
@@ -66,10 +67,15 @@ def __init__(__self__, application_id=None, available_to_other_tenants=None, hom
     """
     > This content is derived from https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/d/azuread_application.html.markdown.
     """
+    pulumi.log.warn("get_application is deprecated: The azure/ad module has moved to a dedicated Pulumi azuread package")
     __args__ = dict()
 
     __args__['name'] = name
     __args__['objectId'] = object_id
+    if opts is None:
+        opts = pulumi.ResourceOptions()
+    if opts.version is None:
+        opts.version = utilities.get_version()
```

Comment:
why did this change?

Reply:
This is the first time we are rolling out the version pinning code for Python - it's just a side effect of upgrading to pulumi-terraform@master